src/main/java/gov/acwi/wqp/etl/wqpNemiNwisCrosswalk/WqpNemiNwisCrosswalkProcessor.java

```diff
@@ -0,0 +1,36 @@
+package gov.acwi.wqp.etl.wqpNemiNwisCrosswalk;
+
+import org.springframework.batch.item.ItemProcessor;
+
+public class WqpNemiNwisCrosswalkProcessor implements ItemProcessor<WqpNemiNwisCrosswalk, WqpNemiNwisCrosswalk> {
+
+	@Override
+	public WqpNemiNwisCrosswalk process(WqpNemiNwisCrosswalk source) throws Exception {
+		WqpNemiNwisCrosswalk result = new WqpNemiNwisCrosswalk();
+		String nemiUrl = null;
+		long methodId = source.getMethodId();
+		String methodType = source.getMethodType();
+		
+		result.setAnalyticalProcedureSource(source.getAnalyticalProcedureSource());
+		result.setAnalyticalProcedureId(source.getAnalyticalProcedureId());
+		result.setCitationMethodNum(source.getCitationMethodNum());
+		result.setMethodId(methodId);
+		result.setWqpSource(source.getWqpSource());
+		result.setMethodType(methodType);
+		
+		switch (methodType) {
```

Comment:
In the big things to come category - the switch will have all new functionality in java 12 and beyond, making this code "cleaner"...